core/org.openjdk.jmc.flightrecorder/src/main/java/org/openjdk/jmc/flightrecorder/stacktrace/tree/AggregatableFrame.java

```diff
@@ -0,0 +1,138 @@
+/*
```

Comment:
This one is from my graph work. I think it is fine to commit it separately here though. I can take a separate pass in my general cleanup and make it a shared class.

Reply:
:+1: Yes, I also copied `Node` from there and changed it slightly. It would be nice to consolidate once both implementations are in.